modules/tracing/mule-tracer-exporter-configuration-api/src/main/java/module-info.java

```diff
@@ -4,7 +4,15 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
+
+/**
+ * Api for the internal tracer exporter configuration.
+ *
+ * @moduleGraph
+ * @since 4.5
```

Comment:
4.6

Reply:
Actually that `module-info` is in `4.5`, I just added the javadoc.